js/runtime.go

```diff
@@ -192,6 +192,14 @@ func (r *Runtime) registerStandardLibray() {
 	}`)
 	r.Set("clearTimeout", clearTimeout)
 	r.Set("clearInterval", clearTimeout)
+	r.Set("sleep", func(call otto.FunctionCall) otto.Value {
```

Comment:
since you are adding sleep here you can remove another sleep here,
https://github.com/skydive-project/skydive/blob/master/api/server/workflow_runtime.go#L62

Reply:
Done!
Thanks @masco-enovance!